src/utils/ModelUtils.js

```diff
@@ -258,6 +258,15 @@ export function addServerInModel(server, model, props) {
   return model.updateIn(['inputModel', 'servers'], list => list.push(fromJS(new_server)));
 }
 
+export function removeServerFromModel(server, model) {
+  return model.updateIn(
+    ['inputModel', 'servers'], list => list.filter(
+      svr => {
+        return svr.get('uid') ? svr.get('uid') !== server.uid : svr.get('id') !== server.id;
```

Comment:
The first clause here, `svr.get('uid')`, is redundant. You could also use `filterNot` to make the logic a little simpler, i.e.:
`['inputModel', 'servers'], list => list.filterNot(svr => svr.get('uid') === server.uid || svr.get('id') === server.id);`

Reply:
will update